toolkit/tools/specreader/specreader.go

```diff
@@ -63,13 +66,23 @@ func main() {
 		logger.Log.Panicf("Value in --workers must be greater than zero. Found %d", *workers)
 	}
 
-	err := parseSPECsWrapper(*buildDir, *specsDir, *rpmsDir, *srpmsDir, *distTag, *output, *workerTar, *workers, *runCheck)
+	var toolchainRPMs []string
+	var err error
+	toolchainManifest := *toolchainManifest
```

Comment:
I understand the shadowing but I would recommend using a slightly different name to avoid confusion.

Reply:
Done